crates/stackable-versioned-macros/src/codegen/vstruct/mod.rs

```diff
@@ -319,22 +319,40 @@ impl VersionedStruct {
         crd_fn_calls: Vec<TokenStream>,
         enum_variants: Vec<(Ident, String)>,
     ) -> TokenStream {
-        let ident = &self.idents.kubernetes;
+        let enum_ident = &self.idents.kubernetes;
+        let enum_vis = &self.visibility;
 
-        let version_enum_definition = self.generate_kubernetes_version_enum(enum_variants);
+        let mut enum_variant_matches = TokenStream::new();
+        let mut enum_variant_idents = TokenStream::new();
+
+        for (enum_variant_ident, enum_variant_display) in enum_variants {
+            enum_variant_idents.extend(quote! {#enum_variant_ident,});
+            enum_variant_matches.extend(quote! {
+                #enum_ident::#enum_variant_ident => f.write_str(#enum_variant_display),
+            });
```

Comment:
**nit**: These appear to only be for Display impl match arms, so we should name it accordingly.

```suggestion
            enum_display_impl_matches.extend(quote! {
                #enum_ident::#enum_variant_ident => f.write_str(#enum_variant_display),
            });
```

Reply:
Yes, makes sense. I will rename locally, because the suggestion doesn't include the `enum_display_impl_matches` declaration.

Reply:
Renamed in c04023a